src/modules/common/MediaDisplay.tsx

```diff
@@ -163,7 +163,7 @@ export function MediaDisplay({ media }: { media: IMedia }) {
   const { t } = useTranslation('modules/common-media-cards');
 
   return (
-    <Group mr="xs" align="stretch" noWrap style={{ maxHeight: 200 }}>
+    <Group mr="xs" align="stretch" noWrap style={{ maxHeight: 200 }} spacing="xs">
```

Comment:
Extract to ``createStyles``

Reply:
- Better readability
- ``style=`` does not access any properties, thus it can be extracted